src/services/langflow_file_service.py

```
@@ -510,50 +542,56 @@ async def upload_and_ingest_file(
+            )
             logger.debug(
-                "[LF] Upload completed successfully",
+                "[LF] Docling upload completed successfully",
                 extra={
-                    "file_id": upload_result.get("id"),
-                    "file_path": upload_result.get("path"),
+                    "task_id": task_id,
+                    "filename": filename,
                 },
             )
         except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)